cmd/nerdctl/container/container_inspect_linux_test.go

```diff
@@ -263,7 +263,6 @@ func TestContainerInspectHostConfig(t *testing.T) {
 		"--read-only",
 		"--shm-size", "256m",
 		"--uts", "host",
-		"--sysctl", "net.core.somaxconn=1024",
```

Comment:
Ideally this should be another sysctl and the inspect result should be validated